udata/templates/macros/metadata.html

```diff
@@ -12,7 +12,7 @@
 <meta name="description" content="{{ description }}" />
 <meta property="og:description" content="{{ description }}" />
 <meta property="og:image" content="{{ image }}" />
-<meta name="keywords" content="{{ ', '.join(current_site.keywords + (keywords)) }}" />
+<meta name="keywords" content="{{ ', '.join(current_site.keywords + keywords) }}" />
```

Comment:
keyword**s** should always be an array. It defaults to an empty array (https://github.com/noirbizarre/udata/blob/python-lints/udata/templates/macros/metadata.html#L5).

If it's not the case, a coma is missing because `(x) = x` and it should be `(keywords, )` but in this case it won't work with list :/